lang/cs/Org.Apache.REEF.Examples/packages.config

```diff
@@ -19,6 +19,6 @@ under the License.
 -->
 <packages>
   <package id="Microsoft.Hadoop.Avro" version="1.5.6" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
```

Comment:
#1126 will need to be updated to update this mention of Newtonsoft.Json to 9.0.1